jac-uk/apply
# Hotfix/1.73.1

## What's included?
Update additional Welsh translation for exercise 192. 

## Who should test?
✅ Product owner
✅ Developers

## How to test?
Example vacancy: https://jac-apply-develop--pr1069-hotfix-1-73-1-hbj7kvqf.web.app/vacancy/Y1gSQnCMhbqpA0nHmDaM/

1. Go to "Location preferences".
2. Check if the question and answers are translated into Welsh correctly.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context

English version

![image](https://github.com/jac-uk/apply/assets/79906532/5a37c97f-6cce-415f-93a9-f734789a0e11)

Welsh version

![image](https://github.com/jac-uk/apply/assets/79906532/a86df708-7f15-4e22-babc-80eb955dd31a)


---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
